CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-03-17
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
```

Comment:
we need to announce deprecation of [List OrchestratorProfile API](https://learn.microsoft.com/en-us/rest/api/aks/container-services/list-orchestrators?view=rest-aks-2019-08-01&tabs=HTTP), it is fully replaced by [List KubernetesVersions API](https://learn.microsoft.com/en-us/rest/api/aks/managed-clusters/list-kubernetes-versions?view=rest-aks-2024-01-01&viewFallbackFrom=rest-aks-2023-11-01&tabs=HTTP)